addon/stories/pix-tooltip.stories.mdx

```diff
@@ -0,0 +1,67 @@
+import { Meta, Story, Canvas, ArgsTable } from '@storybook/addon-docs/blocks';
+import centered from '@storybook/addon-centered/ember';
+import * as stories from './pix-tooltip.stories.js';
+
+<Meta
+  title='Basics/Tooltip'
+  component='PixTooltip'
+  decorators={[centered]}
+  argTypes={stories.argTypes}
+/>
+
+# Pix Tooltip
+
+Une infobulle qui s'affiche au survol d'un élément.
+
+Ce composant est utilisé comme wrapper, c'est à dire qu'il encadre l'élément sur lequel on souhaite ajouter une infobulle.
+
+| | |
+|-|-|
+|`INFO` | A noter que le wraper PixTooltip est en `display: flex;`, il s'adaptera donc à la taille de ses enfants. Ainsi si votre élément ne s'affiche plus comme avant après l'ajout de la PixTooltip, veillez à rajouter les dimensions voulues à l'enfant.
+
+<Canvas>
+  <Story name="Tooltip" story={stories.tooltip} height="160px" />
+</Canvas>
+
+## Usage
+
+```js
+<PixTooltip
+  @text='Hey'
+  @position='top'
+  >
+  <button>Tooltip top</button>
+</PixTooltip>
+
+<br><br>
+
+<PixTooltip
+  @text='Hello'
+  @position='right'
+  >
+  <button>Tooltip right</button>
+</PixTooltip>
+
+<br><br>
+
+<PixTooltip
+  @text='A looooooong looooooong text but in a single line'
+  @position='bottom'
+  @inline=true
+  >
+  <button>Tooltip bottom</button>
+</PixTooltip>
+
+<br><br>
+
+<PixTooltip
+  @text='In accumsan scelerisque sapien, et lacinia nisi efficitur a.'
+  @position='left'
+  >
+  <button>Tooltip left</button>
+</PixTooltip>
+```
```

Comment:
```suggestion
```js
<PixTooltip
  @text='Hey'
  @position='top'
  >
  <button>Tooltip top</button>
</PixTooltip>

<br><br>

<PixTooltip
  @text='A looooooong looooooong text but in a single line'
  @position='bottom'
  @inline=true
  >
  <button>Tooltip bottom</button>
</PixTooltip>
```
```